connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -209,6 +209,15 @@ message WriteStreamOperationStart {
     string path = 11;
     string table_name = 12;
   }
+
+  optional Foreach foreach = 13;
```

Comment:
remove `optional`. These are optional already.

Reply:
I remember some method like `hasForeach` or so will complain if doing that. I will try and get back to you